charts/opentelemetry-kube-stack/examples/cloud-demo/values.yaml

```diff
@@ -0,0 +1,42 @@
+clusterName: demo
+opentelemetry-operator:
+  enabled: true
+defaultCRConfig:
+  presets:
+    otlpExporter:
+      enabled: true
+      endpoint: ingest.example.com:443
+      headers:
+        "access-token": "${ACCESS_TOKEN}"
+collectors:
+  daemon:
+    env:
+      - name: ACCESS_TOKEN
+        valueFrom:
+          secretKeyRef:
+            key: access_token
+            name: otel-collector-secret
+  cluster:
+    env:
```

Comment:
this technically could go in the defaultCRConfig, but I wanted to show that users can override these individually